quarkus/service/build.gradle.kts

```diff
@@ -136,6 +136,7 @@ dependencies {
 
   // override dnsjava version in dependencies due to https://github.com/dnsjava/dnsjava/issues/329
   intTestImplementation(platform(libs.dnsjava))
+  testFixturesImplementation(platform(libs.dnsjava))
```

Comment:
Don't ask me why, but IntelliJ now refuses to import the project without this 🤷‍♂️